lib/active_merchant/billing/gateways/paymill.rb

```diff
@@ -48,6 +48,11 @@ def void(authorization, options={})
       end
 
       def store(credit_card, options={})
+        # The store request requires a currency and amount of at least $1 USD.
+        # This is used for an authorization.
```

Comment:
I don't understand why this comment is talking about authorization within the `store` method. Is the issue that a `store` transaction for PayMill now runs an authorization first before storing the card? Do all `store` methods work that way?

Reply:
Yep, that is the issue. The ```store``` requests for Paymill started resulting in errors when Paymill added ```currency``` and ```amount``` as required fields for the ```store``` operation. I can tweak the comment to be more explicit in saying that the authorization is something Paymill does internally. I'm not aware of any other gateway ```store```s that work this way.